src/App.css

```diff
@@ -1,38 +1,21 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
+:root{
+  --fontSize-desktop-heading: 64px;
+  --fontSize-desktop-subHeading: 50px;
+  --fontSize-desktop-subSubHeading: 30px;
+  --fontSize-desktop-body: 20px;
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
+  --fontSize-mobile-heading: 30px;
+  --fontSize-mobile-subHeading: 20px;
+  --fontSize-mobile-body: 16px;
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  --color-main: #1C307A;
+  --color-img: #7CA6EE;
+  --color-accent: #B2CBF5;
+  --color-btnPrimary: #568CDB;
+  --color-btnSecondary: #6D97D5;
+  --color-background: #F6F6F6;
 }
 
-.App-link {
-  color: #61dafb;
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
-}
+#app{
+  font-family: "Barlow";
```

Comment:
@praneershrest by adding this line, the font will by default be `Barlow` for the entire app, so we dont need to add that to every single CSS file